server/index.js

```diff
@@ -1,18 +1,47 @@
 const express = require('express')
-const app = express();
+const bodyParser = require('body-parser')
+const mongoose = require('mongoose')
+const dotenv = require("dotenv");
+dotenv.config();
+
+const app = express()
 const port = 8000;
+// const { mongoUrl } = process.env.mongoUrl
+
+require('./models/User');
+
+const requireToken = require('./middleware/requireToken')
+app.use(bodyParser.json())
+const authRoutes = require('./routes/authRoutes')
+app.use(authRoutes)
+
+
+mongoose.connect(process.env.mongoUrl, {
+  // dbName: "UserCollection",
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false,
+})
+
+mongoose.connection.on('connected', () => {
+  console.log("Connected to mongo")
+})
+mongoose.connection.on('error', (err) => {
+  console.log("Not Connected to mongo", err)
+})
+
+
+
+// app.post('/', (req, res) => {
+//   console.log(req.body)
+//   res.send('hello')
+// })
+
+app.get('/', requireToken, (req, res) => {
+  res.send("Your email is " + req.user.email)
```

Comment:
Instead we can have a more User-Friendly message: `Welcome to the All-Notes API v1`